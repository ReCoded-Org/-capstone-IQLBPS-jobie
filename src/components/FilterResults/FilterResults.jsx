import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faBookmark } from "@fortawesome/free-regular-svg-icons";
import FilterButton from "./FilterButton";

function FilterResults({ setData, data }) {
  const handleChange = (props) => {
    const newArray = data?.map((key) => ({
      ...key,
      date: new Date(key.postingDate),
    }));
    if (props === "newest") {
      const sortingDataByNewest = newArray.sort((a, b) => b.date - a.date);
      setData(sortingDataByNewest);
    } else {
      const sortingDataByOldest = newArray.sort((a, b) => a.date - b.date);
      setData(sortingDataByOldest);
    }
  };

  return (
    <div className="bg-gray-100 w-full ml-5">
      <div className="flex justify-between">
        <h2 className="ml-5 mt-5 text-lg">
          {" "}
          Total{" "}
          <span className="text-accent mr-1 font-semibold">{data?.length}</span>
          Results
        </h2>
        <FilterButton handleChange={handleChange} />
      </div>
      {data?.length
        ? data.map((job) => {
            return (
              <div
                key={job.id}
                className="rounded-md grid justify-items-center gap-3 m-5 bg-white drop-shadow-xl grid-cols-2 grid-rows-4 md:grid-cols-5 md:grid-rows-2"
              >
                <img
                  className="order-1 col-span-2 h-20  md:row-span-2 md:col-span-1 md:scale-60 object-contain md:h-28 md:order-1"
                  src={job.src}
                  alt="hiring company logo"
                />
                <h2 className="order-4 font-semibold text-center md:font-semibold md:mt-3  md:order-2">
                  {job.position}
                </h2>
                <h2 className="order-5 md:font-semibold md:mt-3 md:order-3">
                  {job.jobModel}
                </h2>
                <h2 className="order-6  font-semibold md:font-semibold md:mt-3 md:order-4 ">
                  ${job.minSalary} - ${job.maxSalary}
                </h2>
                <FontAwesomeIcon
                  className="order-3 md:mt-3 md:order-5"
                  icon={faBookmark}
                />
                <h2 className="order-8 font-semibold mb-5 md:font-normal md:order-6">
                  {" "}
                  {job.employer}
                </h2>
                <h2 className="order-9 mb-5 md:order-7"> {job.jobType}</h2>
                <h2 className="order-7 md:order-8"> {job.payFreq}</h2>
                <h2 className="order-2 font-semibold md:font-normal md:order-9">
                  {" "}
                  {job.postingDate}
                </h2>
              </div>
            );
          })
        : null}
    </div>
  );
}

export default FilterResults;
