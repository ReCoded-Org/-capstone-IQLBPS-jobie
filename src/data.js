export const showcaseData = [
  {
    id: 1,
    header: 'Full Time Jobs',
    body: 'if you are set on your career and want to work full time, then a full time job is just the one for you, clicking here will show you all the current full time jobs in our platform',
    button: 'Full time jobs >',
  },
  {
    id: 2,
    header: 'Part Time Jobs',
    body: 'if you are set on your career and want to work full time, then a full time job is just the one for you, clicking here will show you all the current full time jobs in our platform',
    button: 'Part Time jobs >',
  },
  {
    id: 3,
    header: 'Remote Jobs',
    body: ' you are set on your career and want to work remotes, then a remotes job is just the one for you,  clicking here will show you all the current remotes jobs in our platform',
    button: 'Remote jobs >',
  },
];
