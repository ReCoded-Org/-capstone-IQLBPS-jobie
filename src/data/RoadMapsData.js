export const data = [
  {
    titleH1: "Web Development",
    id: 1,
    RoadMapsData: [
      {
        svg: {
          xmlns: "http://www.w3.org/2000/svg",
          viewBox: "0 0 384 512",
          d: "M0 32l34.9 395.8L191.5 480l157.6-52.2L384 32H0zm308.2 127.9H124.4l4.1 49.4h175.6l-13.6 148.4-97.9 27v.3h-1.1l-98.7-27.3-6-75.8h47.7L138 320l53.5 14.5 53.7-14.5 6-62.2H84.3L71.5 112.2h241.1l-4.4 47.7z",
        },
        link: "https://roadmap.sh/frontend",
        title: "Frontend",
        id: 1,
      },
      {
        svg: {
          xmlns: "http://www.w3.org/2000/svg",
          viewBox: "0 0 512 512",
          d: "M495.9 166.6c3.2 8.7 .5 18.4-6.4 24.6l-43.3 39.4c1.1 8.3 1.7 16.8 1.7 25.4s-.6 17.1-1.7 25.4l43.3 39.4c6.9 6.2 9.6 15.9 6.4 24.6c-4.4 11.9-9.7 23.3-15.8 34.3l-4.7 8.1c-6.6 11-14 21.4-22.1 31.2c-5.9 7.2-15.7 9.6-24.5 6.8l-55.7-17.7c-13.4 10.3-28.2 18.9-44 25.4l-12.5 57.1c-2 9.1-9 16.3-18.2 17.8c-13.8 2.3-28 3.5-42.5 3.5s-28.7-1.2-42.5-3.5c-9.2-1.5-16.2-8.7-18.2-17.8l-12.5-57.1c-15.8-6.5-30.6-15.1-44-25.4L83.1 425.9c-8.8 2.8-18.6 .3-24.5-6.8c-8.1-9.8-15.5-20.2-22.1-31.2l-4.7-8.1c-6.1-11-11.4-22.4-15.8-34.3c-3.2-8.7-.5-18.4 6.4-24.6l43.3-39.4C64.6 273.1 64 264.6 64 256s.6-17.1 1.7-25.4L22.4 191.2c-6.9-6.2-9.6-15.9-6.4-24.6c4.4-11.9 9.7-23.3 15.8-34.3l4.7-8.1c6.6-11 14-21.4 22.1-31.2c5.9-7.2 15.7-9.6 24.5-6.8l55.7 17.7c13.4-10.3 28.2-18.9 44-25.4l12.5-57.1c2-9.1 9-16.3 18.2-17.8C227.3 1.2 241.5 0 256 0s28.7 1.2 42.5 3.5c9.2 1.5 16.2 8.7 18.2 17.8l12.5 57.1c15.8 6.5 30.6 15.1 44 25.4l55.7-17.7c8.8-2.8 18.6-.3 24.5 6.8c8.1 9.8 15.5 20.2 22.1 31.2l4.7 8.1c6.1 11 11.4 22.4 15.8 34.3zM256 336c44.2 0 80-35.8 80-80s-35.8-80-80-80s-80 35.8-80 80s35.8 80 80 80z",
        },
        link: "https://roadmap.sh/backend",
        title: "Backend",
        id: 2,
      },
      {
        svg: {
          xmlns: "http://www.w3.org/2000/svg",
          viewBox: "0 0 640 512",
          d: "M308.5 135.3c7.1-6.3 9.9-16.2 6.2-25c-2.3-5.3-4.8-10.5-7.6-15.5L304 89.4c-3-5-6.3-9.9-9.8-14.6c-5.7-7.6-15.7-10.1-24.7-7.1l-28.2 9.3c-10.7-8.8-23-16-36.2-20.9L199 27.1c-1.9-9.3-9.1-16.7-18.5-17.8C173.7 8.4 166.9 8 160 8s-13.7 .4-20.4 1.2c-9.4 1.1-16.6 8.6-18.5 17.8L115 56.1c-13.3 5-25.5 12.1-36.2 20.9L50.5 67.8c-9-3-19-.5-24.7 7.1c-3.5 4.7-6.8 9.6-9.9 14.6l-3 5.3c-2.8 5-5.3 10.2-7.6 15.6c-3.7 8.7-.9 18.6 6.2 25l22.2 19.8C32.6 161.9 32 168.9 32 176s.6 14.1 1.7 20.9L11.5 216.7c-7.1 6.3-9.9 16.2-6.2 25c2.3 5.3 4.8 10.5 7.6 15.6l3 5.2c3 5.1 6.3 9.9 9.9 14.6c5.7 7.6 15.7 10.1 24.7 7.1l28.2-9.3c10.7 8.8 23 16 36.2 20.9l6.1 29.1c1.9 9.3 9.1 16.7 18.5 17.8c6.7 .8 13.5 1.2 20.4 1.2s13.7-.4 20.4-1.2c9.4-1.1 16.6-8.6 18.5-17.8l6.1-29.1c13.3-5 25.5-12.1 36.2-20.9l28.2 9.3c9 3 19 .5 24.7-7.1c3.5-4.7 6.8-9.5 9.8-14.6l3.1-5.4c2.8-5 5.3-10.2 7.6-15.5c3.7-8.7 .9-18.6-6.2-25l-22.2-19.8c1.1-6.8 1.7-13.8 1.7-20.9s-.6-14.1-1.7-20.9l22.2-19.8zM208 176c0 26.5-21.5 48-48 48s-48-21.5-48-48s21.5-48 48-48s48 21.5 48 48zM504.7 500.5c6.3 7.1 16.2 9.9 25 6.2c5.3-2.3 10.5-4.8 15.5-7.6l5.4-3.1c5-3 9.9-6.3 14.6-9.8c7.6-5.7 10.1-15.7 7.1-24.7l-9.3-28.2c8.8-10.7 16-23 20.9-36.2l29.1-6.1c9.3-1.9 16.7-9.1 17.8-18.5c.8-6.7 1.2-13.5 1.2-20.4s-.4-13.7-1.2-20.4c-1.1-9.4-8.6-16.6-17.8-18.5L583.9 307c-5-13.3-12.1-25.5-20.9-36.2l9.3-28.2c3-9 .5-19-7.1-24.7c-4.7-3.5-9.6-6.8-14.6-9.9l-5.3-3c-5-2.8-10.2-5.3-15.6-7.6c-8.7-3.7-18.6-.9-25 6.2l-19.8 22.2c-6.8-1.1-13.8-1.7-20.9-1.7s-14.1 .6-20.9 1.7l-19.8-22.2c-6.3-7.1-16.2-9.9-25-6.2c-5.3 2.3-10.5 4.8-15.6 7.6l-5.2 3c-5.1 3-9.9 6.3-14.6 9.9c-7.6 5.7-10.1 15.7-7.1 24.7l9.3 28.2c-8.8 10.7-16 23-20.9 36.2L315.1 313c-9.3 1.9-16.7 9.1-17.8 18.5c-.8 6.7-1.2 13.5-1.2 20.4s.4 13.7 1.2 20.4c1.1 9.4 8.6 16.6 17.8 18.5l29.1 6.1c5 13.3 12.1 25.5 20.9 36.2l-9.3 28.2c-3 9-.5 19 7.1 24.7c4.7 3.5 9.5 6.8 14.6 9.8l5.4 3.1c5 2.8 10.2 5.3 15.5 7.6c8.7 3.7 18.6 .9 25-6.2l19.8-22.2c6.8 1.1 13.8 1.7 20.9 1.7s14.1-.6 20.9-1.7l19.8 22.2zM464 400c-26.5 0-48-21.5-48-48s21.5-48 48-48s48 21.5 48 48s-21.5 48-48 48z",
        },
        link: "https://javascript.plainenglish.io/the-complete-full-stack-web-developer-roadmap-2021-7495b4ae59ac",
        title: "Full Stack",
        id: 3,
      },
    ],
  },
  {
    titleH1: "Design",
    id: 2,
    RoadMapsData: [
      {
        svg: {
          xmlns: "http://www.w3.org/2000/svg",
          viewBox: "0 0 512 512",
          d: "M352 96c0 14.3-3.1 27.9-8.8 40.2L396 227.4c-23.7 25.3-54.2 44.1-88.5 53.6L256 192h0 0l-68 117.5c21.5 6.8 44.3 10.5 68.1 10.5c70.7 0 133.8-32.7 174.9-84c11.1-13.8 31.2-16 45-5s16 31.2 5 45C428.1 341.8 347 384 256 384c-35.4 0-69.4-6.4-100.7-18.1L98.7 463.7C94 471.8 87 478.4 78.6 482.6L23.2 510.3c-5 2.5-10.9 2.2-15.6-.7S0 501.5 0 496V440.6c0-8.4 2.2-16.7 6.5-24.1l60-103.7C53.7 301.6 41.8 289.3 31.2 276c-11.1-13.8-8.8-33.9 5-45s33.9-8.8 45 5c5.7 7.1 11.8 13.8 18.2 20.1l69.4-119.9c-5.6-12.2-8.8-25.8-8.8-40.2c0-53 43-96 96-96s96 43 96 96zm21 297.9c32.6-12.8 62.5-30.8 88.9-52.9l43.7 75.5c4.2 7.3 6.5 15.6 6.5 24.1V496c0 5.5-2.9 10.7-7.6 13.6s-10.6 3.2-15.6 .7l-55.4-27.7c-8.4-4.2-15.4-10.8-20.1-18.9L373 393.9zM256 128c17.7 0 32-14.3 32-32s-14.3-32-32-32s-32 14.3-32 32s14.3 32 32 32z",
        },
        link: "https://bootcamp.uxdesign.cc/the-self-taught-ui-ux-designer-roadmap-in-2021-aa0f5b62cecb",
        title: "UI/UX",
        id: 1,
      },
      {
        svg: {
          xmlns: "http://www.w3.org/2000/svg",
          viewBox: "0 0 512 512",
          d: "M512 256c0 .9 0 1.8 0 2.7c-.4 36.5-33.6 61.3-70.1 61.3H344c-26.5 0-48 21.5-48 48c0 3.4 .4 6.7 1 9.9c2.1 10.2 6.5 20 10.8 29.9c6.1 13.8 12.1 27.5 12.1 42c0 31.8-21.6 60.7-53.4 62c-3.5 .1-7 .2-10.6 .2C114.6 512 0 397.4 0 256S114.6 0 256 0S512 114.6 512 256zM128 288c0-17.7-14.3-32-32-32s-32 14.3-32 32s14.3 32 32 32s32-14.3 32-32zm0-96c17.7 0 32-14.3 32-32s-14.3-32-32-32s-32 14.3-32 32s14.3 32 32 32zM288 96c0-17.7-14.3-32-32-32s-32 14.3-32 32s14.3 32 32 32s32-14.3 32-32zm96 96c17.7 0 32-14.3 32-32s-14.3-32-32-32s-32 14.3-32 32s14.3 32 32 32z",
        },
        link: "https://www.codelivly.com/graphic-designer-roadmap-2022/",
        title: "Graphics",
        id: 2,
      },
      {
        svg: {
          xmlns: 'http://www.w3.org/2000/svg"',
          viewBox: "0 0 576 512",
          d: "M316.9 18C311.6 7 300.4 0 288.1 0s-23.4 7-28.8 18L195 150.3 51.4 171.5c-12 1.8-22 10.2-25.7 21.7s-.7 24.2 7.9 32.7L137.8 329 113.2 474.7c-2 12 3 24.2 12.9 31.3s23 8 33.8 2.3l128.3-68.5 128.3 68.5c10.8 5.7 23.9 4.9 33.8-2.3s14.9-19.3 12.9-31.3L438.5 329 542.7 225.9c8.6-8.5 11.7-21.2 7.9-32.7s-13.7-19.9-25.7-21.7L381.2 150.3 316.9 18z",
        },
        link: "https://enlightmenow.com/learn-logo-design-as-beginner/",
        title: "Logo",
        id: 3,
      },
      {
        svg: {
          xmlns: "http://www.w3.org/2000/svg",
          viewBox: "0 0 512 512",
          d: "M418.4 157.9c35.3-8.3 61.6-40 61.6-77.9c0-44.2-35.8-80-80-80c-43.4 0-78.7 34.5-80 77.5L136.2 151.1C121.7 136.8 101.9 128 80 128c-44.2 0-80 35.8-80 80s35.8 80 80 80c12.2 0 23.8-2.7 34.1-7.6L259.7 407.8c-2.4 7.6-3.7 15.8-3.7 24.2c0 44.2 35.8 80 80 80s80-35.8 80-80c0-27.7-14-52.1-35.4-66.4l37.8-207.7zM156.3 232.2c2.2-6.9 3.5-14.2 3.7-21.7l183.8-73.5c3.6 3.5 7.4 6.7 11.6 9.5L317.6 354.1c-5.5 1.3-10.8 3.1-15.8 5.5L156.3 232.2z",
        },
        link: "https://github.com/eovchar/neteng-roadmap",
        title: "Network",
        id: 4,
      },
      {
        svg: {
          xmlns: "http://www.w3.org/2000/svg",
          viewBox: "0 0 512 512",
          d: "M256 0c4.6 0 9.2 1 13.4 2.9L457.7 82.8c22 9.3 38.4 31 38.3 57.2c-.5 99.2-41.3 280.7-213.7 363.2c-16.7 8-36.1 8-52.8 0C57.3 420.7 16.5 239.2 16 140c-.1-26.2 16.3-47.9 38.3-57.2L242.7 2.9C246.8 1 251.4 0 256 0zm0 66.8V444.8C394 378 431.1 230.1 432 141.4L256 66.8l0 0z",
        },
        link: "https://pauljerimy.com/security-certification-roadmap/",
        title: "Security",
        id: 5,
      },
      {
        svg: {
          xmlns: "http://www.w3.org/2000/svg",
          viewBox: "0 0 640 512",
          d: "M144 480C64.5 480 0 415.5 0 336c0-62.8 40.2-116.2 96.2-135.9c-.1-2.7-.2-5.4-.2-8.1c0-88.4 71.6-160 160-160c59.3 0 111 32.2 138.7 80.2C409.9 102 428.3 96 448 96c53 0 96 43 96 96c0 12.2-2.3 23.8-6.4 34.6C596 238.4 640 290.1 640 352c0 70.7-57.3 128-128 128H144zm79-217c-9.4 9.4-9.4 24.6 0 33.9s24.6 9.4 33.9 0l39-39V392c0 13.3 10.7 24 24 24s24-10.7 24-24V257.9l39 39c9.4 9.4 24.6 9.4 33.9 0s9.4-24.6 0-33.9l-80-80c-9.4-9.4-24.6-9.4-33.9 0l-80 80z",
        },
        link: "https://www.simplilearn.com/cloud-computing-career-guide-pdf",
        title: "Cloud",
        id: 6,
      },
    ],
  },
];
