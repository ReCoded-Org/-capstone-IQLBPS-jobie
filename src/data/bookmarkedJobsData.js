export const bookmarkedJobsData = [{
        id: "1",
        src: "/assets/filterResults/Murex.png",
        position: "DevOps Engineer",
        jobModel: "Remote",
        employer: "Murex",
        jobType: "Full Time",
        status: "closed",
    },
    {
        id: "2",
        src: "/assets/filterResults/Murex.png",
        position: "Front-end Developer",
        jobModel: "Remote",
        employer: "Murex",
        jobType: "Part-time",
        status: "open",
    },
    {
        id: "3",
        src: "/assets/filterResults/EAB.png",
        position: "Associate Data Scientist",
        jobModel: "Remote",
        employer: "EAB",
        jobType: "Part Time",
        status: "open",
    },
];