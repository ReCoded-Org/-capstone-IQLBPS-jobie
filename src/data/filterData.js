export const filterData = [
  {
    src: "/assets/filterResults/Murex.png",
    position: "DevOps Engineer",
    jobModel: "Remote",
    minSalary:"1000",
    maxSalary:"5000",
    employer: "Murex",
    jobType: "Full Time",
    payFreq: "Monthly",
    postingDate: "07/12/2022",
    location: "Iraq",
    id: "1",
  },
  {
    src: "/assets/filterResults/Murex.png",
    position: "DevOps Engineer",
    jobModel: "Remote",
    minSalary:"4000",
    maxSalary:"6000",
    employer: "Murex",
    jobType: "Internship",
    payFreq: "Monthly",
    postingDate: "06/26/2022",
    location: "Lebanon",
    id: "2",
  },
  {
    src: "/assets/filterResults/EAB.png",
    position: "Associate Data Scientist",
    jobModel: "Remote",
    minSalary:"2500",
    maxSalary:"4500",
    employer: "EAB",
    jobType: "Part Time",
    payFreq: "Monthly",
    postingDate: "06/19/2022",
    location: "Iraq",
    id: "3",
  },
  {
    src: "/assets/filterResults/EAB.png",
    position: "Data Analyst",
    jobModel: "Remote",
    minSalary:"6500",
    maxSalary:"8000",
    employer: "EAB",
    jobType: "Full Time",
    payFreq: "Monthly",
    postingDate: "06/19/2022",
    location: "Syria",
    id: "4",
  },
  {
    src: "/assets/filterResults/EAB.png",
    position: "Network Engineer",
    jobModel: "Remote",
    minSalary:"5000",
    maxSalary:"8000",
    employer: "EAB",
    jobType: "Full Time",
    payFreq: "Monthly",
    postingDate: "06/19/2022",
    location: "Yemen",
    id: "5",
  },
  {
    src: "/assets/filterResults/EAB.png",
    position: "Creative Project Manager",
    jobModel: "Remote",
    minSalary:"5000",
    maxSalary:"6000",
    employer: "EAB",
    jobType: "Full Time",
    payFreq: "Monthly",
    postingDate: "01/01/2022",
    location: "Lebanon",
    id: "6",
  },
];
