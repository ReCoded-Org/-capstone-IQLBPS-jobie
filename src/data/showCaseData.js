export const showCaseData = {
  companies: [
    "facebook",
    "amazon",
    "microsoft",
    "apple",
    "google",
    "airbnb",
    "spotify",
    "yahoo",
    "netflix",
  ],
};
