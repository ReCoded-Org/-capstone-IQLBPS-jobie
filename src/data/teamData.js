export const teamMembers = [
   {"name":"Allan Ahmed",
     "position":"Frontend Developer",
     "src":"/assets/meetOurTeam/allan.jpg",
     "linkedin":"https://www.linkedin.com/in/allan-ahmed/"
    },
    {"name":"Iman Zuher",
     "position":"Frontend Developer",
     "src":"/assets/meetOurTeam/iman.jpg",
     "linkedin":"https://www.linkedin.com/in/iman-zuher/"
    },
    {"name":"Ahmed Rashid",
     "position":"Frontend Developer",
     "src":"/assets/meetOurTeam/ahmed.jpg",
     "linkedin":"https://www.linkedin.com/in/ahmed-rashid-8a9b9116b/"
    },
    {"name":"Noora Hakim",
     "position":"Frontend Developer",
     "src":"/assets/meetOurTeam/noora.jpg",
     "linkedin":"https://www.linkedin.com/in/noora-hakim-156144244/"
    },
    {"name":"Hakam Shakir",
     "position":"Frontend Developer",
     "src":"/assets/meetOurTeam/hakam.jpg",
     "linkedin":"https://www.linkedin.com/in/hakam-shakir-31b358204/"
    },
    {"name":"Marwa Salameh",
     "position":"Frontend Developer",
     "src":"/assets/meetOurTeam/marwa.jpg",
     "linkedin":"https://www.linkedin.com/in/marwa-salameh/"
    },
    {"name":"Zakaria Ahmad",
     "position":"Frontend Developer",
     "src":"/assets/meetOurTeam/zakaria.jpg",
     "linkedin":"https://www.linkedin.com/in/zakaria-ahmad-76754a239/"
   },
   {"name":"Parykhan Jameel",
     "position":"Frontend Developer",
     "src":"/assets/meetOurTeam/parykhan.jpg",
     "linkedin":"https://www.linkedin.com/in/parykhan-jameel-5b5078241/"
   },
   {"name":"Lawen Sardar",
   "position":"Frontend Developer",
   "src":"/assets/meetOurTeam/lawen.jpg",
   "linkedin":"https://www.linkedin.com/in/lawen-sardar/"
  }
]