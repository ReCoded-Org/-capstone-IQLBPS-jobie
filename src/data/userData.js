export const userData = {
      src: "/assets/userProfile/Dunkin.png",
      name: "Dunkin"
    }
  