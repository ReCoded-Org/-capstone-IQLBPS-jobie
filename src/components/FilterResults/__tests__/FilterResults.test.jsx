import renderer from "react-test-renderer";
import FilterResults from "../FilterResults";
import { filterData } from "../../../data/filterData";

it("renders FilterResults Component correctly", () => {
  const tree = renderer
    .create(<FilterResults filterData={filterData} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
