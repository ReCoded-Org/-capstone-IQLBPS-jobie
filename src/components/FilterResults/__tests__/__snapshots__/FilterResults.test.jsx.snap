// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FilterResults Component correctly 1`] = `
<div
  className="bg-gray-100 w-full ml-5"
>
  <div
    className="flex justify-between"
  >
    <h2
      className="ml-5 mt-5 text-lg"
    >
       
      Total
       
      <span
        className="text-accent mr-1 font-semibold"
      />
      Results
    </h2>
    <div
      className="relative inline-block text-left"
      data-headlessui-state=""
      data-testid="filter-button"
    >
      <div>
        <button
          aria-expanded={false}
          aria-haspopup={true}
          className="rounded-full bg-gray-300 mr-5 mt-5 pl-5 pr-5 text-lg duration-500 hover:bg-accent"
          data-headlessui-state=""
          data-testid="filter-botton-toggle"
          id="headlessui-menu-button-:r0:"
          onClick={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          type="button"
        >
           
          Sort results by
        </button>
      </div>
    </div>
  </div>
</div>
`;
